www/gatsby-node.js

```diff
@@ -14,9 +14,9 @@ require(`dotenv`).config({
   path: `.env.${process.env.NODE_ENV}`,
 })
 
-if (process.env.NODE_ENV === `production` && !process.env.GITHUB_API_TOKEN) {
+if (!process.env.GITHUB_API_TOKEN) {
   throw new Error(
-    `A GitHub token is required to build the site. Check the README.`
+    `A GitHub token is required to build the site. Check www/README.md.`
```

Comment:
@amberleyromo built this so the starter library wouldn’t crash local dev for someone working on something that's _not_ the starter library.

I think that's a good pattern, because getting a GitHub token for an unrelated part of the site you're working on feels like a decent-sized hurdle.

Maybe we can log a better warning about _why_ a GitHub token is required, and let people know they can safely ignore if they're not working on the starter library?

Reply:
I was writing out exactly this comment, thank you @jlengstorf 👍

Reply:
@jlengstorf @amberleyromo this fails for me regardless of which section I'm working on. Does it not for you?

See https://github.com/gatsbyjs/gatsby/issues/5073#issuecomment-426073720